modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -45,7 +46,6 @@ Before you register {micro} clients to your {productname} Server, you need to ad
 
 include::snippets/arch-other-note.adoc[]
 
-ifeval::[{suma-content} == true]
```

Comment:
As mentioned, SLE Micro comes from SCC for Uyuni as well (but not the client tools). A subscription is required.

Reply:
ok